src/components/BagItems.jsx

```diff
@@ -0,0 +1,123 @@
+import { useDispatch, useSelector } from "react-redux";
+import { Flex, Text, Button, Heading } from "@sparrowengg/twigs-react";
+import { setTotalAmount } from "../redux/actions";
+import { useNavigate } from "react-router-dom";
+import { useEffect, useMemo } from "react";
+const BagItems = () => {
+  const cart = useSelector((state) => state.cart);
+  const totalAmount = useSelector((state) => state.totalAmount);
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    let total = 0;
+    cart.forEach((ele) => {
+      total = total + Number(ele.cartCount) * Number(ele.price);
+    });
+    total = Number(total.toFixed(3));
+    dispatch(setTotalAmount(total));
+  }, [cart]);
+  return (
+    <>
+      <Flex
+        flexDirection="column"
+        justifyContent="space-between"
+        gap="$5"
+        css={{ width: "300px", overflow: "scroll", height: "100%" }}
+      >
+        <Heading
+          size="h2"
+          css={{
+            marginBlock: "$15",
+            textAlign: "center",
+          }}
+        >
+          My Bag
+        </Heading>
+
+        <Flex
+          css={{ flexWrap: "wrap", overflow: "scroll" }}
```

Comment:
@arun-121 by default if we set to scroll means the scrollbar would be there right?

Reply:
changed to scroll auto